index.html

```diff
@@ -0,0 +1,320 @@
+              <button type="button" class="btn-xxl">See Project</button>
+            </div>
+          </div>
+        </div>
+
+      <div class="card-layout">
+        <a href="#">
+          <img
+            class="image-fluid"
+            src="images/works/image_4.png"
+            alt="background_image"
+          />
+        </a>
+        <div class="p-title">
+          <h3 class="title">Tonic</h3>
+          <ul class="flex-items">
+            <li class="list-item canopy">CANOPY</li>
+            <li class="list-item role">Back End Dev</li>
+            <li class="list-item year">2015</li>
+          </ul>
+          <p class="text_paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="btn-container-flex">
+            <li class="btn">
+              <button class="btn-sm" type="button">html</button>
+            </li>
+            <li class="btn">
+              <button class="btn-md" type="button">css</button>
+            </li>
+            <li class="btn">
+              <button class="btn-lg" type="button">javaScript</button>
+            </li>
+          </ul>
+          <div class="btn-container">
+            <button type="button" class="btn-xxl">See Project</button>
+          </div>
+        </div>
+        </div>
+      <div class="card-layout">
+        <a href="#">
+          <img
+            class="image-fluid"
+            src="images/works/image_3.png"
+            alt="background_image"
+          />
+        </a>
+        <div class="p-title">
+          <h3 class="title">Tonic</h3>
+          <ul class="flex-items">
+            <li class="list-item canopy">CANOPY</li>
+            <li class="list-item role">Back End Dev</li>
+            <li class="list-item year">2015</li>
+          </ul>
+          <p class="text_paragraph">
+            Multi-Post Stories.
+          </p>
+          <ul class="btn-container-flex">
+            <li class="btn">
+              <button class="btn-sm" type="button">html</button>
+            </li>
+            <li class="btn">
+              <button class="btn-md" type="button">css</button>
+            </li>
+            <li class="btn">
+              <button class="btn-lg" type="button">javaScript</button>
+            </li>
+          </ul>
+          <div class="btn-container">
+            <button type="button" class="btn-xxl">See Project</button>
+          </div>
+        </div>
+     
+      </div>
+    </div>
+    </section>
+
+    <section class="about">
+      <h2 class="about-title">About<br />Myself<br /></h2>
+      <p class="paragraph">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <span class="connect">LETS CONNECT</span>
```

Comment:
- [ ] I think it would be better to wrap the "LETS CONNECT" action word in a heading tag(h2-h6). This gives more semantic meaning to the action word 😄 . 
e.g `<h3 class="connect">LETS CONNECT</h3>`

Oftentimes, users skim page content to find the information they came for. The right headers make it easy for your visitors to scan the page and decide on which sections to read 🤓